src/sentry/api/endpoints/organization_auditlogs.py

```diff
@@ -40,7 +41,10 @@ def get(self, request: Request, organization) -> Response:
             queryset = queryset.filter(actor=query["actor"])
 
         if "event" in query:
```

Comment:
@mbauer404 it looks like this should be 
```suggestion
        if "event" in query and query["event"] is not None:
```

Reply:
@mbauer404 I wanted to double check to ensure the suggestion I gave makes sense; would you expect `event` to be required in `query`?

Reply:
@dashed Thank you! The "event" is only expected if the audit log entries are being filtered using the drop down menu in the UI, and `if "event" in query:` is needed to only filter the queryset when that drop down is used. If the "event" is not in the query, no additional filtering of the query is needed.